examples/hat/binaryclock.py

```diff
@@ -0,0 +1,35 @@
+import unicornhat as uh
+import datetime
+
```

Comment:
You should include a `print("""""")` here to explain what people should expect from your example. IE:

```python
print("""Binary Clock

Displays Hours, Minutes, Seconds, Day and Month
in a binary format.
""")
```